### PR TITLE
EKSA Version Documentation

### PR DESCRIPTION
**Pull request by tatlat**
*Issue #, if available:*

*Description of changes:*

*Testing (if applicable):*

*Documentation added/planned (if applicable):*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/content/en/docs/clustermgmt/cluster-upgrades/baremetal-upgrades.md [hunk 1]
```diff
@@ -39,6 +39,10 @@ We encourage that you stay up to date with the latest EKS Anywhere version, as n
 
 Download the [latest or target EKS Anywhere release](https://github.com/aws/eks-anywhere/releases/) and run `eksctl anywhere upgrade cluster` command to upgrade a cluster to a specific EKS Anywhere version.
 
+Workload clusters can also be upgraded via the API (`kubectl apply` or GitOps) by changing `bundlesRef` for EKS Anywhere version 0.16.5 or below. Starting from v0.17.0, `bundlesRef` can be set to null and `eksaVersion` can be used instead.
+
+`bundlesRef` is a reference to a bundles resource on the cluster whereas `eksaVersion` must be a valid SemVer value that maps to an EKSARelease resource on the cluster (e.g. vX.X.X-prerelease+build -> vX-X-X-prerelease-plus-build). The workload's version may not exceed the management cluster. Any upgrades to `eksaVersion` must also be sequential relative to minor version. Both of these fields are automatically updated by EKS Anywhere and only need to be manually changed when upgrading via the API.
```
Comment (csplinter) [inline docs/content/en/docs/clustermgmt/cluster-upgrades/baremetal-upgrades.md:44]: We need to explicitly mention patch version upgrades. Can users skip patch versions?

Comment (csplinter) [inline docs/content/en/docs/clustermgmt/cluster-upgrades/baremetal-upgrades.md:44]: What is "prerelease+build" and "prerelease-plus-build" ?

Comment (tatlat) [reply docs/content/en/docs/clustermgmt/cluster-upgrades/baremetal-upgrades.md]: Yes, they can. Mentioned it.

Comment (tatlat) [reply docs/content/en/docs/clustermgmt/cluster-upgrades/baremetal-upgrades.md]: I added extra details. Does it make sense now?

**Review by csplinter — APPROVED** [review]

**Review by drewvanstone — APPROVED** [review]

**Pull request merged**